www/wml/Portfile

```diff
@@ -46,6 +46,7 @@ depends_build           port:lynx
 depends_lib-append      port:gdbm \
                         port:gettext \
                         port:libiconv \
+                        port:libpng \
```

Comment:
If you add a library dependency, you must increase the port's revision.

Reply:
Since you are increasing the revision anyway, this is a good time to fix the gettext dependency. A [library dependency on gettext is no longer correct](https://lists.macports.org/pipermail/macports-dev/2021-December/044041.html).